core/src/main/java/com/yahoo/ycsb/generator/UnixEpochTimestampGenerator.java

```diff
@@ -0,0 +1,188 @@
+/**
+ * Copyright (c) 2016 Yahoo! Inc. All rights reserved.
```

Comment:
Instead of Yahoo! Inc. This should be `Copyright (c) 2016 YCSB contributors. All rights reserved.`

Same goes for the test file.